frontend/admin/src/js/components/pool/EditPool/editPoolOperations.graphql

```diff
@@ -142,11 +142,13 @@ mutation updatePoolAdvertisement(
 
 mutation publishPoolAdvertisement($id: ID!) {
   publishPoolAdvertisement(id: $id) {
+    id
```

Comment:
nice, i like the idea of always returning the ID 👍  seems like this was just a lone straggler, and that's the default :)